imports/plugins/included/shipping-shippo/register.js

```diff
@@ -31,6 +35,12 @@ Reaction.registerPackage({
     provides: ["shippingSettings"],
     container: "connection",
     template: "shippoSettings"
+  }, {
+    label: "Shippo",
```

Comment:
I don't think you need a separate entry here. You should just be adding this to the `provides` array

Reply:
@zenweasel There is some code that extracts "shippo" from the registry's `name`. This is how we get "avalara" from Avalara's registry. So this entry is needed.